### ISSUE TITLE
CAS on/off filters should share common settings

### ISSUE THREAD

**Issue by jmaferreira** [keeps/roda#660]
The goal is to simplify the roda-wui.properties

Suggestion: 
Activate via parameters on the web.xml that would set the settings prefix.

**Comment by rui-castro** [comment]
@jmaferreira @hsilva-keep @luis100 done in PR #698

Configuration on roda-wui.properties changed to this form:
```bash
##############################################
# Internal filters
##############################################

ui.filter.internal.enabled = false
ui.filter.internal.realm = RODA REST API
ui.filter.internal.exclusions = ^/swagger.json,^/v1/theme/?

##############################################
# CAS filters
##############################################

ui.filter.cas.enabled = true
ui.filter.cas.casServerUrlPrefix = https://localhost:8443/cas
ui.filter.cas.casServerLoginUrl = https://localhost:8443/cas/login
ui.filter.cas.exclusions = ^/swagger.json,^/v1/theme/?,^/v1/auth/ticket?
# RODA base address
ui.filter.cas.serverName = http://localhost:8888
ui.filter.cas.exceptionOnValidationFailure = false
ui.filter.cas.redirectAfterValidation = false
```

In web.xml the parameter `config-prefix` was added to all filters. Like these:
```xml
<filter>
	<filter-name>InternalWebAuthFilter</filter-name>
	<filter-class>org.roda.wui.filter.OnOffFilter</filter-class>
	<init-param>
		<param-name>inner-filter-class</param-name>
		<param-value>org.roda.wui.filter.InternalWebAuthFilter</param-value>
	</init-param>
	<init-param>
		<param-name>config-prefix</param-name>
		<param-value>ui.filter.internal</param-value>
	</init-param>
</filter>
```

```xml
<filter>
	<filter-name>CASAuthenticationFilter</filter-name>
	<filter-class>org.roda.wui.filter.OnOffFilter</filter-class>
	<init-param>
		<param-name>inner-filter-class</param-name>
		<param-value>org.jasig.cas.client.authentication.AuthenticationFilter</param-value>
	</init-param>
	<init-param>
		<param-name>config-prefix</param-name>
		<param-value>ui.filter.cas</param-value>
	</init-param>
	<!-- ... -->
</filter>
```

**Comment by luis100** [comment]
Checked and merged into lf_dev at 2f5f819a66c6ee90954bb430b2845beeca4621c4
